docs/source/getting-started/tabular-data/index.rst

```diff
@@ -0,0 +1,103 @@
+The `User Guide <https://ec.europa.eu/eurostat/documents/1978984/6037342/EULFS-Database-UserGuide.pdf>`_ describes many variables. Our examples will use just a few. (Descriptions are copied from the User Guide.) 
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Variable
+     - Definition
+     - Coding
+   * - ``SEX``
+     - Sex
+     - 1: Male; 2: Female
+   * - ``AGE``
+     - Age of the Individual During the Reference Week
+     - Single Years
+   * - ``ILOSTAT``
+     - Labour Status During the Reference Week
+     - 1: Did any work for pay or profit during the reference week - one hour or more (including family workers but excluding conscripts on compulsory military or community service); 2: Was not working but had a job or business from which he/she was absent during the reference week (including family workers but excluding conscripts on compulsory military or community service); 3: Was not working because of lay-off; 4: Was a conscript on compulsory military or community service; 5: Other (15 years or more) who neither worked nor had a job or business during the reference week; 9: Not applicable (child less than 15 years old)
+   * - ``HWUSUAL``
+     - Number of Hours Per Week Usually Worked
+     - 00: Usual hours cannot be given because hours worked vary considerably from week to week or from month to month; 01-98: Number of hours usually worked in the main job; 99: Not applicable; blank: No answer
+   * - ``QUARTER``
+     - Fixed Reference Quarter
+     - Single Quarter
+   * - ``YEAR``
+     - Fixed Reference Year
+     - Single Year
+
+Compositor Overview
+-------------------
+The compositor is the foundation of our differentially private queries. It essentially takes in our data and our specifications for the queries that we would like to run. At this point, we won't be directly referencing our data again and we could theoretically delete it! 
+
+.. code-block:: python
+
+   context = dp.Context.compositor(
+       data=df,
+       privacy_unit=dp.unit_of(contributions=36),
+       privacy_loss=dp.loss_of(epsilon=1.0),
+       split_evenly_over=10,
+       margins={
+           ("YEAR", ): dp.Margin(max_partition_length=60_000_000, max_partition_contributions=4),
+           ("YEAR", "QUARTER",): dp.Margin(max_partition_length=60_000_000, max_partition_contributions=1),
+           (): dp.Margin(max_partition_length=60_000_000),
+       },
+   )
+
+**Parameters**:
+
+* *privacy_unit:* How many rows each individual or entity of interest contributes to our data frame. In this case, we are analyzing the data from across 13 years and each year has 4 quarters. Therefore, the unit of privacy is 36. If we were to analyze a particular quarter in a particular year, the unit of privacy would be 1 since each individual would be represented once. 
```

Comment:
This is quite unclear for multiple reasons. How do the numbers 13 and 4 imply the unit to be 36? Also, what is the calculation for the quarterly stuff you mention in the final sentence?

Reply:
mistyped, its 9 years! the data just ends in 2013